packages/fast-components-react-msft/README.md

```diff
@@ -0,0 +1,32 @@
+# FAST Components React Microsoft
+A set of React components which implements the Microsoft styling.

```

Comment:
Maybe we should be explicit about which style language?

Reply:
We are not being explicit in the package name right now even though we are following the fluent styling, the idea is that we can just change the style to whatever new styling direction emerges, whether its fluent or something else.

Reply:
Got it